scoresheets/GPSR.tex

```diff
@@ -2,13 +2,14 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[3]{250}{Perform each task}
-	\penaltyitem[3]{50}{Using custom operator or bypassing ASR}
-
-	\scoreheading{Bonus rewards}
-	\scoreitem[3]{100}{Understand command given by naive operator}
-	\scoreitem[3]{100}{Provide audio recording and transcript}
-	\scoreitem{150}{Autonomously leaving the arena}
+	\scoreitem[3]{250}{Complete a command}
+	\penaltyitem[3]{50}{Receiving command not from the instruction point}
+	\penaltyitem[3]{50}{Custom Operator}
+	\penaltyitem[3]{75}{Alternative HRI}
+	
+	\scoreheading{Bonus Rewards}
+	\scoreitem[3]{50}{Understand command given by naive operator}
+	\scoreitem{100}{Autonomously leaving the arena after completing all 3 commands}
```

Comment:
I've never really understood the reasoning for this bonus; if the robot was able to complete three arbitrary commands, it can surely navigate to a waypoint outside the arena. Would rather see these points put on something more interesting; either more points for the naive operator or something new.

Reply:
+1

Reply:
The reward is not for being able to leave, but to be able to solve 3 commands in time. Leaving just signifying that the bonus goal has been achieved. So the bonus is more for being really fast. It's difficult to make bonuses for GPSR since you don't know which commands will be generated.

Reply:
Ah, I see. It makes sense when explained. I would suggest lowering the bar ever so slightly then to being exactly that; "completing all 3 commands."

Reply:
Changed bonus